src/main/resources/templates/layouts/baseLayout.html

```diff
@@ -7,34 +7,38 @@
   <title layout:title-pattern="$CONTENT_TITLE"></title>
   <!--[if lte IE 8]>
   <link
-      href="@{{cdnUrl}/stylesheets/govuk-frontend/v2.12.0/govuk-frontend-ie8-2.12.0.min.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
-      rel="stylesheet" type="text/css"/><![endif]-->
-
+      href="@{{cdnUrl}/stylesheets/govuk-frontend/v3.11.0/govuk-frontend-ie8-3.11.0.min.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
+      rel="stylesheet"
+      type="text/css"/>
+  <![endif]-->
   <link
       media="all"
       rel="stylesheet"
-      th:href="@{{cdnUrl}/stylesheets/govuk-frontend/v2.12.0/govuk-frontend-2.12.0.min.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
+      th:href="@{{cdnUrl}/stylesheets/govuk-frontend/v3.11.0/govuk-frontend-3.11.0.min.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
       type="text/css"/>
-
-  <link rel="stylesheet" th:href="@{/css/application.css}"
+  <link rel="stylesheet"
+        th:href="@{/css/application.css}"
         type="text/css"/>
-
   <link
       rel="icon"
       th:href="@{{cdnUrl}/images/favicon.ico?0.17.3(cdnUrl=${@environment.getProperty('cdn.url')})}"
       type="image/png"/>
-
   <link
       rel="icon"
       th:href="@{{cdnUrl}/images/royal-coat-of-arms-white.png(cdnUrl=${@environment.getProperty('cdn.url')})}"
       type="image/png"/>
-
+  <script th:inline="javascript">
+        window.SERVICE_NAME = 'lfp'
```

Comment:
Service name needs to change here please

Reply:
Changed.